### PR TITLE
Add configurable timeouts for the cybersource rest api calls

### PR DESCRIPTION
**Pull request by cpennington**

### CODE REVIEW

**Review by mulby — APPROVED** [review]
Generally looks good to me, wondering if we could empirically determine the values we set in production?

File: ecommerce/extensions/payment/processors/cybersource.py [hunk 1]
```diff
@@ -695,6 +695,12 @@ class CybersourceREST(Cybersource):  # pragma: no cover
 
     NAME = "cybersource-rest"
 
+    def __init__(self, site):
+        super().__init__(site)
+        configuration = self.configuration
```
Comment (mulby) [inline ecommerce/extensions/payment/processors/cybersource.py]: nit: this doesn't appear to be used

Comment (cpennington) [reply ecommerce/extensions/payment/processors/cybersource.py]: Oops. Fixed.

**Comment by cpennington** [comment]
Yes, I just realized that we can look at newrelic to find existing call times. Also, that pointed out that we're also hitting the rest api for the capture context, so I just added the timeout to those.

Our current NR data doesn't show any requests longer than 700ms. I adjusted the defaults to give us a fair amount of headroom, but we can tune them down once we start having more data.

**Review by inventhouse — APPROVED** [review]

**Pull request merged**